### PR TITLE
add documentation to proto files

### PR DESCRIPTION
**Pull request by JonasVautherin**
Bring documentation from C++ to the proto files for a few plugins (action, calibration, camera, param, telemetry).

The other plugins will follow in a new PR soon.

@hamishwille: Those comments are the ones being parsed by #98 and used to generate language-specific documentation in the source files. I'll open a PR for the corresponding python templates as soon as those PRs are merged.

### CODE REVIEW

File: protos/action/action.proto [hunk 1]
```diff
@@ -84,11 +160,11 @@ message SetTakeoffAltitudeResponse {
 message GetMaximumSpeedRequest {}
 message GetMaximumSpeedResponse {
     ActionResult action_result = 1;
-    float speed = 2;
+    float speed = 2; // Maximum speed (in metres/second)
```
Comment (julianoes) [inline protos/action/action.proto]: Not a change for this PR but we should decide on what we do about units!

Comment (JonasVautherin) [reply protos/action/action.proto]: What do you mean? In terms of how we document them?

Comment (julianoes) [reply protos/action/action.proto]: I think they should be part of the variable name (unless we can have units that reflect it).

Comment (JonasVautherin) [reply protos/action/action.proto]: Right. I don't think I have a strong opinion on that matter, but if we want to change, a good moment would be after we have the C++ auto-generation, and before we move to version 1.x :+1:.

**Pull request merged**